CONTRIBUTING.md

```diff
@@ -352,4 +352,4 @@ differently.
 We strive to follow the guidelines provided by
 [kubernetes/charts](https://github.com/kubernetes/charts/blob/master/REVIEW_GUIDELINES.md)
 and the [Helm chart best practices
-guide](https://github.com/kubernetes/helm/tree/master/docs/chart_best_practices).
```

Comment:
Thanks this is certainly better than a 404!

We may need to make more updates down the line to not end up confusing that this is made during helm2 era rather than the new helm3 era which the current helm docs have entered without retaining all of their old docs.